src/RageSound.cpp

```diff
@@ -499,33 +496,25 @@ int RageSound::GetSourceFrameFromHardwareFrame( std::int64_t iHardwareFrame, boo
  */
 float RageSound::GetPositionSeconds( bool *bApproximate, RageTimer *pTimestamp ) const
 {
-	/* Get our current hardware position. */
-	std::int64_t iCurrentHardwareFrame = SOUNDMAN->GetPosition( pTimestamp );
-
-	/* Lock the mutex after calling SOUNDMAN->GetPosition().  We must not make driver
-	 * calls with our mutex locked (driver mutex < sound mutex). */
-	LockMut( m_Mutex );
-
-	if( bApproximate )
-		*bApproximate = false;
+	// Get our current hardware position.
+	std::int64_t iCurrentHardwareFrame = SOUNDMAN->GetPosition(pTimestamp);
+	float f_sampleRate = samplerate();
```

Comment:
`fSampleRate` is the correct naming. Also just static cast here like it was being done previously so you don't have to do it elsewhere.